src/msgwindow.c

```diff
@@ -488,6 +519,32 @@ void msgwin_status_add(const gchar *format, ...)
 	}
 }
 
+/**
+ * Logs a formatted status message *without* setting the status bar.
+ *
+ * Use @ref ui_set_statusbar() to display text on the statusbar.
+ *
+ * @param format @c printf()-style format string.
+ * @param ...    Arguments for the @c format string.
+ *
+ * @see msgwin_status_add_string()
+ *
+ * @since 0.15
```

Comment:
Same

Reply:
Seems to be 0.12 actually :]
Or drop it, because it wasn't in the original and nobody actually cares